jsapp/js/account/add-ons/oneTimeAddOnRow.component.tsx

```diff
@@ -1,14 +1,14 @@
 import styles from 'js/account/add-ons/addOnList.module.scss';
 import React, {useMemo, useState} from 'react';
-import {Organization, Product} from 'js/account/stripe.types';
+import type {Organization, Product} from 'js/account/stripe.types';
 import KoboSelect3 from 'js/components/special/koboAccessibleSelect';
-import {KoboSelectOption} from 'js/components/common/koboSelect';
+import type {KoboSelectOption} from 'js/components/common/koboSelect';
 import BillingButton from 'js/account/plans/billingButton.component';
 import {postCheckout} from 'js/account/stripe.api';
 import {useDisplayPrice} from 'js/account/plans/useDisplayPrice.hook';
```

Comment:
The display price for the NLP package shows up as being a monthly fee, although it's a one-time purchase:
![image](https://github.com/kobotoolbox/kpi/assets/7819986/0cb692a0-3198-4684-a302-6ac03e49784f)

This one's more on me, I think; I forgot to account for one-time prices when writing `useDisplayPrice`. Would you mind updating it to omit "USD/month" for prices that aren't recurring? Recurring prices should have a `price.recurring.interval`.